flask_mailgun.py

```diff
@@ -101,27 +179,33 @@ def send(self, **data):
         from_ = data.pop('from_', None)
         if from_ is None:
             from_ = 'no-reply@{}'.format(self.domain)
+
         data['from'] = from_
+
         if 'html' in data and 'text' not in data:
             data['text'] = html2text(data['html'])
+
         if self.debug:
             data.setdefault('html', '(no HTML)')
             current_app.logger.debug(debug_template, data)
             return
-        url = 'https://api.mailgun.net/v3/{}/messages'.format(self.domain)
```

Comment:
Yeah, could be many reasons. Mailgun changing the URL on their end suddenly? Overriding / aliasing the Mailgun URL using local DNS in your environment? Setting up a "fake Mailgun" for test purposes in your environment? But really, the main reason (in my opinion) is that a URL should always be configurable somehow, simply as a matter of principle, it shouldn't be hard-coded into a library like this with no way of overriding it.